frontend/src/views/Seedlot/SeedlotRegistrationForm/index.tsx

```diff
@@ -4,28 +4,115 @@ import {
   FlexGrid,
   Row,
   Breadcrumb,
-  BreadcrumbItem
+  BreadcrumbItem,
+  Button
 } from '@carbon/react';
+import { ArrowRight, CheckmarkOutline } from '@carbon/icons-react';
 
 import PageTitle from '../../../components/PageTitle';
 import SeedlotRegistrationProgress from '../../../components/SeedlotRegistrationProgress';
 import OrchardStep from '../../../components/SeedlotRegistrationSteps/OrchardStep';
-import InterimStorage from '../../../components/SeedlotRegistrationProgress/InterimStorage';
+import InterimStorage from '../../../components/SeedlotRegistrationSteps/InterimStep';
 import OwnershipStep from '../../../components/SeedlotRegistrationSteps/OwnershipStep';
-
+import { AllStepData } from './definitions';
+import {
+  initInterimState,
+  initOrchardState,
+  initOwnershipState
+} from './utils';
+import { SingleOwnerForm } from '../../../components/SeedlotRegistrationSteps/OwnershipStep/definitions';
 import './styles.scss';
+import InterimForm from '../../../components/SeedlotRegistrationSteps/InterimStep/definitions';
+import { SeedlotOrchard } from '../../../types/SeedlotTypes/SeedlotOrchard';
+
+const defaultCode = '16';
+const defaultAgency = '0032 - Strong Seeds Orchard - SSO';
+const defaultPayment = 'ITC - Invoice to client address';
+const agencyOptions = [
+  '0032 - Strong Seeds Orchard - SSO',
+  '0035 - Weak Seeds Orchard - WSO',
+  '0038 - Okay Seeds Orchard - OSO',
+  '0041 - Great Seeds Orchard - GSO',
+  '0043 - Bad Seeds Orchard - BSO'
+];
```

Comment:
Maybe moving these consts to a different file?

Reply:
If we are gonna get these from API calls maybe this is fine? these will be replaced by live data from DB